### ISSUE TITLE
Idea: extend Hydrant Type quest to allow change to emergency=dry_riser_inlet

### ISSUE THREAD

**Issue by pkoby** [streetcomplete/StreetComplete#3637]
I saw a discussion somewhere a couple months back, I don't remember where (possibly OSM Slack) about fire hydrants. It was thought by one commenter that probably a lot of "fire hydrants" in OSM are actually Dry Riser Inlets/Fire Dept. Connections (FDC) (tagged `emergency=dry_riser_inlet`). Instead of providing outflow of water, they are an attachment point into the building to direct water from a hydrant into upper levels.

In my experience, these are always on walls of larger buildings, and without knowing what FDC stands for, I've tagged many as hydrants. Taginfo shows ~1000 `emergency=dry_riser_inlet`, and ~3400 `emergency=fire_hydrant` + `fire_hydrant:type=wall` (about ~230000 are untagged for type). Many of those may be Dry Riser Inlets.

My thought was that in the Hydrant Type quest, there could be an option in Other Answers to say it's a Dry Riser Inlet, and perhaps also a note to disambiguate the wall type. Alternatively, there could simply be a fifth option (to Pillar, Underground, Wall, and Pond) to add Dry Riser Inlet, even though it's not actually a hydrant (and just change the tag). This may be helpful for the many who wouldn't know the difference, but does mess up the symmetry.

**Comment by timothywashere** [comment]
If people don't know what a Dry Riser is, they would probably mistag it as a Wall Hydrant, so having Dry Riser as an extra option next to Wall Hydrant on the hydrant type would make them think about it at least and try to spot the difference.

**Comment by westnordost** [comment]
Doesn't necessarily have to be an other answer. If it is an other answer, users would need to know about such things **before** looking in the other answers menu. If it is possible to distinguish them clearly visually from wall hydrants, there could just be a picture they can tap on.

Is it possible to distinguish them easily? Do you have (free) pictures of such dry riser inlets? Are they distinguishable by anything **other** than them being labelled that way?

**Comment by westnordost** [comment]
It looks like they are called "standpipes". "dry risers" are standpipes that are dry. But there are also "wet standpipes". Are they tagged the same? Couldn't they also be mistaken for wall hydrants?

See https://en.wikipedia.org/wiki/Standpipe_(firefighting)

**Comment by pkoby** [comment]
Fair point, I don't think most people would know about them to check the Other Answers.

I can take photos when I next see some. I was just looking up the same resources you just posted. I have not personally knowingly seen a wet standpipe, but based on a web search, they're also labeled as such.

I don't think OSM has a robust tagging scheme for these differentiations. There is https://wiki.openstreetmap.org/wiki/Tag:emergency%3Dstandpipe and https://wiki.openstreetmap.org/wiki/Tag:emergency%3Dsiamese, but they're contributed by one user, and not highly used.

**Comment by westnordost** [comment]
If there is not accepted tagging, then it must first be proposed in the wiki before it could be used by StreetComplete.

The issue is that `dry_riser_inlet` is already approved while `standpipe` is not. But there is an obvious overlap between the two (the former is a subcategory of the latter). An overlap we noticed in 5min of browsing the thing on wikipedia but apparently noone of the 23 people who voted yes [on the proposal](https://wiki.openstreetmap.org/wiki/Proposed_features/Dry_riser_inlet) noticed... 🙄

**Comment by pkoby** [comment]
Yeah, I hadn't delved too far into the non-OSM-Wiki research prior to this. That's my fault...

**Comment by westnordost** [comment]
Soo, I'll close this for now. Feel free to create a new issue when the tagging issue has been raised on the wiki/ml/...

**Comment by Helium314** [comment]
[`emergency=fire_service_inlet`](https://wiki.openstreetmap.org/wiki/Tag:emergency%3Dfire_service_inlet) appears to be a replacement for `emergency=dry_riser_inlet` to address the dry/wet issue.
